scripts/set-version.sh

```
@@ -73,7 +73,8 @@ update_app_stream_version() {
 
 update_ios_marketing_version() {
   local file_path=$1
-  local new_version=$2
+  # Remove everything after the "-"
+  local new_version=$(echo "$2" | sed -E 's/-.*$//')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)